azure_batch/main.tf

```
@@ -0,0 +1,183 @@
+/*
+
+# Purpose
+
+Create an Azure Batch account [1] with a pool populated of Docker-compatible compute nodes [2].
+
+# Notes
+
+1. Create a storage account [3] to store files used by the compute nodes to perform any backend 
+   calculation.
+2. Create a container within the above-mentioned storage account with a user-tunable retention 
+   policy of n days [4].
+3. Create a private container registry [5] and push a private Docker image in that registry.
+4. Create the Batch account. Warning: it is not possible to create more then a single Batch Account
+   per subscription and per region due to quota limitations (see Ref. [6]). Consider increasing that
+   if needed.
```

Would put the warning on a new line so it's more visible:
```suggestion
4. Create the Batch account.
   Warning: it is not possible to create more then a single Batch Account per subscription and per
   region due to quota limitations [6]. Consider increasing that if needed.
```